src/BaseCommand.ts

```diff
@@ -34,17 +32,25 @@ export abstract class BaseCommandClass {
     }
     // Save optional arguments
     if (options) {
-      const { isActive, name, matchMode, requireActiveGame, includeHelpCommand } = options;
+      const { isActive, name, matchMode, requireActiveGame, includeHelpCommand, allowedMessageTypes } = options;
       if (name) this.name = name;
-      if (isActive) this.isActive = isActive;
+      if (isActive !== undefined) this.isActive = isActive;
       if (matchMode) this.matchMode = matchMode;
-      if (requireActiveGame) this.requireActiveGame = requireActiveGame;
-      if (includeHelpCommand) this.includeHelpCommand = includeHelpCommand;
+      if (requireActiveGame !== undefined) this.requireActiveGame = requireActiveGame;
```

Comment:
Im imagining you have 3 options for the boolean fields
* Send in true
* Send in false
* Dont send it

what I wanted to achieve here was so "Send in false" would work, because that didn't work before because false is not truthy

Reply:
Given the current set of compiler flags one could do e.g. `requireActiveGame = null` as argument when specifiying the `options` and that would be valid input. Then we would assign `null` here. Not sure if it could cause any problems.

I see your point regarding supplying `false` as the argument and the above comment regarding potentially changed default values. Again adding the _strict_ compileflag will solve this in the long run.

Reply:
https://github.com/Lolispo/inhouseBot/issues/25